README.md

```diff
@@ -1,6 +1,37 @@
 # Buildstamp monorepo
+Utility for generating buildstamp, a file with build-time info.
```

Comment:
build-time?
Utility for generating **buildstamp** file, which contains build meta info like gitcommit hash, timestamp, repo name and so on. This file could be a part of some release artifact (npm-package, docker-image) and makes it self-descriptive.